Update troubleshooting.md (#112)
R2Northstar/NorthstarWiki: docs/installing-northstar/troubleshooting.md

## File Corruption Detected <a href="#file-corruption" id="file-corruption"></a>

![Engine Error: File corruption detected. Please repair or re-install the game.](https://user-images.githubusercontent.com/39478251/147338706-74797220-7d7f-4c81-9ba0-d88e29a2a1e2.png)

> Don't panic! This warning isn't as serious as it seems. If you get this warning after launching the game, try updating your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases), as this error was mostly resolved in version `v1.4.0`.

Reviewer: ```suggestion
Don't panic! This warning isn't as serious as it seems. It's simply an incorrect error message caused by Origin/EA App. If you get this warning after launching the game, try updating your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases), as this error was mostly resolved in version `v1.4.0`.
```

> If that doesn't work, you should verify your Titanfall 2 files. If you're confused on how to verify files, follow [this](troubleshooting.md#verify-files) guide.

Reviewer: ```suggestion
If that doesn't work, you should verify your Titanfall2 files. If you're confused on how to verify files, follow [this](troubleshooting.md#verify-files) guide.
```

## File Corruption Detected <a href="#file-corruption" id="file-corruption"></a>

![Engine Error: File corruption detected. Please repair or re-install the game.](https://user-images.githubusercontent.com/39478251/147338706-74797220-7d7f-4c81-9ba0-d88e29a2a1e2.png)

> Don't panic! This warning seems to be caused by Origin and none of your files are actually corrupted. If you get this warning after launching the game try updating your Northstar install to the [Newest Release](https://github.com/R2Northstar/Northstar/releases), as this error was resolved in version 1.4.0.

Reviewer: ```suggestion
Don't panic! This warning seems to be caused by Origin and none of your files are actually corrupted. If you get this warning after launching the game try updating your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases), as this error was resolved in version `v1.4.0`.
```

Reviewer 2: Fixed the capital letters as well as making the v1.4.0 stand out

* For Origin
  * `Your library > Right click TF|2 > Game Properties > Advanced Launch Options > Command Line Arguments > -northstar or -vanilla`

## Verifying Game Files <a href="#verify-files" id="verify-files"></a>

This is a small guide to help you understand how to verify the files of your game

> * For steam

Reviewer: ```suggestion
* For Steam
```

  * `Your library > Right click TF|2 > Properties > Local Files > Verify integrity of game files...`
* For EA app
  * `My collection > Click the three dots on TF|2 > Repair`
* For Origin
  * `My Game Library > Click on TF|2 > Click the gear icon > Repair`

## Access Violation

If your error says `Access Violation | Attempted to read from 0x00000000` specifically DO NOT post just THAT.\
The real error is most likely slightly higher. Please post that in issues or the discord

## I can't play the Campaign <a href="#campaign" id="campaign"></a>

> If having trouble playing the campaign, update your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases)

Reviewer: ```suggestion
If you're having trouble playing the campaign, update your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases) as this issue was resolved in `v1.11.2` of Northstar.
```

## I can't play the Campaign <a href="#campaign" id="campaign"></a>

If having trouble playing the campaign, try the following fixes

> 1. Update your Northstar install to the [Newest Release](https://github.com/R2Northstar/Northstar/releases)

Reviewer: ```suggestion
1. Update your Northstar install to the [newest release](https://github.com/R2Northstar/Northstar/releases)
```

> 2. Add `-vanilla` to your launch options and play vanilla Titanfall

Reviewer: We should just drop the `-vanilla` mention cause that literally just launches vanilla Titanfall2 without any Northstar modifications. At this point, one can just launch vanilla Titanfall2 directly :P

Reviewer 2: I'm not too sure how github works and to concretely mention changes I've made but I'll do it through this for now: Vanilla mention has been dropped, as well as the unnecessary 1. following the information to update
